### ISSUE TITLE
About mean and std for normalization

### ISSUE THREAD

**Issue by MSAE-demo** [LightDXY/GreedyFool#3]
In the `nips_gd.py`, the normalization for the input image is:
```python
mean_arr = (0.5,0.5,0.5)
stddev_arr = (0.5,0.5,0.5)
im_size = 299
test_dataset = McDataset(
        opt.dataroot,
        transform=transforms.Compose([
            transforms.ToTensor(),
            transforms.Normalize(mean_arr, stddev_arr)
        ]),
)
```

However, the normalization for the pretrained inceptionv3 model shown in pytorch.org is:
```python
preprocess = transforms.Compose([
    transforms.Resize(299),
    transforms.CenterCrop(299),
    transforms.ToTensor(),
    transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
])
input_tensor = preprocess(input_image)
```

Could you explain why the mean and std is different?

**Comment by LightDXY** [comment]
Please refer to the model architecture in torchvision, it renormalizes the mean and std from [0.485...] to [0.5...] in the forward. (https://pytorch.org/vision/stable/_modules/torchvision/models/inception.html#inception_v3,  function '_transform_input')
Here I just remove the renormalizes in the forward and normalize it to the correct mean and std directly.
